ArcGISRuntimeSDKQt_QMLSamples/Routing/DisplayRouteLayer/DisplayRouteLayer.qml

```diff
@@ -0,0 +1,156 @@
+// [WriteFile Name=DisplayRouteLayer, Category=Routing]
+// [Legal]
+// Copyright 2022 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// [Legal]
+
+import QtQuick 2.12
+import Esri.ArcGISRuntime 100.15
+import QtQuick.Controls 2.12
+import QtQuick.Layouts 1.11
+
+Rectangle {
+    id: rootRectangle
+    clip: true
+    width: 800
+    height: 600
+
+    property var featureCollection: null
+    property var featureCollectionLayer: null
```

Comment:
optional, but you can strongly type these, which should be a slight optimization for QML, and also gives you intellisense for those types 

```suggestion
    property FeatureCollection featureCollection: null
    property FeatureCollectionLayer featureCollectionLayer: null
```